src/Microsoft.Graph.Core/Requests/BaseRequest.cs

```diff
@@ -40,6 +41,9 @@ public class BaseRequest : IBaseRequest
 
             this.RequestUrl = this.InitializeUrl(requestUrl);
 
+            this.sdkVersionHeaderName = CoreConstants.Headers.SdkVersionHeaderName;
```

Comment:
Should these be changeable at run-time. If not, I'd make them const in this class.